pkg/server/status_local_file_retrieval.go

```diff
@@ -97,6 +118,34 @@ func stacksLocal(req *serverpb.StacksRequest) (*serverpb.JSONResponse, error) {
 	return &serverpb.JSONResponse{Data: buf.Bytes()}, nil
 }
 
+// FilterStacksWithLabels writes all the stacks that have pprof labels matching
+// any one of the expectedLabels to resBuf.
+func FilterStacksWithLabels(stacks []byte, labelFilter string) []byte {
```

Comment:
This is tested from `pprofui`, which seems right. So why not move it there, too? Is this the reason why you switched the pprofui tests to `pprofui_test`? If that's the case definitely move it and consider a switch back to `pprofui` for the test file.

Reply:
good point, moved to pprofui and moved the test file back to the original package.